src/markdown-pages/docs/cli-command-reference/runs-get.md

```diff
@@ -101,6 +132,7 @@ start-time     : 2023-05-05 06:00:14
 end-time       : 2023-05-05 06:00:15
 duration(ms)   : 1000
 test-name      : dev.galasa.Zos3270LocalJava11Ubuntu
+requestor      : bobsmith 
 bundle         : dev.galasa
 run-log        : https://127.0.0.1/ras/run/cbd-123/runlog
 
```

Comment:
I think we should have an example using --result with the comma separated list e.g 
```galasactl runs get --age 1d --result failed,envfail```